doc/stdlib/strings.rst

```diff
@@ -54,10 +54,18 @@
 
    Convert a string to a contiguous ASCII string (all characters must be valid ASCII characters).
 
+.. function:: ascii(::Ptr{UInt8}, [length])
+
+   Create an ASCII string from the address of a C (0-terminated) string encoded in ASCII. A copy is made; the ptr can be safely freed. If ``length`` is specified, the string does not have to be 0-terminated.
+
 .. function:: utf8(::Array{UInt8,1})
 
    Create a UTF-8 string from a byte array.
 
+.. function:: ascii(::Ptr{UInt8}, [length])
```

Comment:
duh!  Where's my dunce cap again!?!  (don't know why that wasn't added to Unicode emoji!)